### DISCUSSION TITLE
Faster processing of video

### DISCUSSION THREAD

**Discussion by xXJohnConstantinexX** [WyattBlue/auto-editor#535] [1. Help]
Faster processing of video and voice analysis using more CPU cores

When running auto-editor this would allow faster cutting and re-encoding of the video files

This should be useful for call users, one example that an 835 MB video (mp4) took 3415.74 seconds (0:56:56)

Running version 25.2.0

**Comment by WyattBlue** [comment]
You can make auto-editor generally faster by using a better computer.
Alternatively use a video codec that is faster to encode (usually at the cost of greater file size).

**Reply by xXJohnConstantinexX** [reply]
Ouch, 
![image](https://github.com/user-attachments/assets/fcf2be09-67c5-4ac8-abb7-b2d617e7bfec)
my system specs.
I used this command:
auto-editor "C:\Users\Alpha\Documents\Session 01.mp4" --margin 0.2sec
The original file size was 835 MB and the newly cut and encoded file is 718 MB. When I was watching the "Analyzing audio volume", and subsequent tasks only 1 of my 64 cores were doing anything (100%).  I just figured I would ask for a feature enhancement. I will read more of the documentation and change the codec to see if that makes any difference, I do think multi-threading is a valid option, imo. 
![image](https://github.com/user-attachments/assets/649be73b-4f7e-4b11-9329-88d53b51abd0)

**Comment by FurkanGozukara** [comment]
I have 32 threads and it is not utilizing all how can I make? @WyattBlue thank you

It shows ETA over 30 minutes for 78 minutes video, 13900K CPU and RTX 3090 TI  - I am using latest FFMPEG 

![image](https://github.com/user-attachments/assets/de843da6-601f-4531-a4f3-d4c4a2255fa8)

![image](https://github.com/user-attachments/assets/27e06265-11ec-44cf-b005-e60c5e71a161)
